src/node/miner.cpp

```diff
@@ -121,7 +121,7 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
     pblocktemplate->vTxFees.push_back(-1); // updated at end
     pblocktemplate->vTxSigOpsCost.push_back(-1); // updated at end
 
-    LOCK2(cs_main, m_mempool.cs);
+    LOCK(::cs_main);
```

Comment:
In the commit message you claim that this is a behaviour change, but I don't see an obvious user facing (or even test-facing) behaviour change. Maybe explain this a bit better or mention that this does *not* change behaviour?

Reply:
Ah yeah true... Guess I meant a "code-behaviour" change, but yes, will update!

Reply:
Fixed as of faddefe547cad0e4ba0f1cab635e48b42e153a11